test/set/set_builder_test.dart

```diff
@@ -130,6 +130,21 @@ void main() {
 
     // Set.
 
+    test('has a method like Set.length', () {
+      expect(new SetBuilder<int>([1, 2]).length, 2);
+      expect(new BuiltSet<int>([1, 2]).toBuilder().length, 2);
+      
+      expect(new SetBuilder<int>([]).length, 0);
+      expect(new BuiltSet<int>([]).toBuilder().length, 0);
+    });
+
+    test('has methods like Set.isEmpty and Set.isNotEmpty', () {
```

Comment:
Please split into separate methods for isEmpty and isNotEmpty as you had for List.